src/java/edu/slu/tpen/entity/image/Canvas.java

```diff
@@ -179,11 +180,11 @@ public static JSONArray getLinesForProject(Integer projectID, String canvasID, I
         String dateString = "";
         String annoListID = getRbTok("SERVERURL") + "project/" + projectID + "/annotations/" + folioNumber;
         annotationList.element("@id", annoListID);
-        annotationList.element("@type", "sc:AnnotationList");
```

Comment:
Prefixes stay in presentation 2.  In presentation 3, this is an `AnnotationPage` .